public/index.css

```diff
@@ -0,0 +1,9 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+html{
+font-family: 'Bitter', serif;
+font-family: 'Montserrat', sans-serif;
```

Comment:
This should probably be one font for the `body` element, one for `h1, h2, ....`, right?